### ISSUE TITLE
Proposition: Change Wins, Draws, and Losses.

### ISSUE THREAD

**Issue by PhantomLel** [Leek727/ChessSchedule#19]
I think that we should have a score variable - 1 for win, 0.5 for draw, and 0 for loss - and have the be the main column in the leaderboard. Then the extra information could be in an expandable columns because the score and Elo would give you an idea of how well a person is doing with just a quick glance.

**Comment by clayrock78** [comment]
I had a similar idea and actually implemented a function as part of the player class called eval_score, which is exactly as you described but `loss = -1` instead of `loss = 0`. What gets put on the leaderboard really doesn't matter THAT much, does it?

**Comment by PhantomLel** [comment]
I think loss should be 0 but we can discuss in class I guess. I think score is good just to get a quick idea of who is doing well. Also I thought that score was going to determine who won?

**Comment by clayrock78** [comment]
I think this is where things are a bit out of our scope and we should ask the client how they want it to be organized. Of course, we can make suggestions but at the end of the day, it's not our choice.

**Comment by clayrock78** [comment]
@PhantomLel 

Do we want to close this issue?
